packages/data-provider/src/services/submissionService.ts

```diff
@@ -371,6 +371,27 @@ const service = (dependencies: BaseDependencies) => {
 				processedEntities: entitiesToProcess,
 			};
 		},
+
+		deleteActiveSubmissionById: async (submissionId: number): Promise<Submission | undefined> => {
+			const { getSubmissionById, update } = submissionRepository(dependencies);
+
+			const submission = await getSubmissionById(submissionId);
+			if (_.isEmpty(submission) || !submission.dictionaryId)
+				throw new BadRequest(`Submission '${submissionId}' not found`);
+
+			// list of considered Open statuses of a submission able to be closed
+			const openStatuses = [SUBMISSION_STATUS.OPEN, SUBMISSION_STATUS.VALID, SUBMISSION_STATUS.INVALID];
+
+			if (!_.includes(openStatuses, submission.status)) {
```

Comment:
Lodash is not necessary here
```suggestion
			if (!openStatuses.includes(submission.status)) {
```

Reply:
@joneubank your golden baby weak support of Array methods is giving us trouble again. halp!